mpa/cls/trainer.py

```diff
@@ -153,10 +151,13 @@ def train_worker(gpu, dataset, cfg, distributed, validate, timestamp, meta):
 
         # updated to adapt list of dataset for the 'train'
         data_loaders = []
+        max_num_images = 0
         for ds in dataset:
             if isinstance(ds, list):
-                sub_loaders = [
-                    build_dataloader(
+                for sub_ds in ds:
+                    if len(sub_ds) > max_num_images:
```

Comment:
max_num_images = max(max_num_images, len(sub_ds))

Reply:
Thanks, that much better